cds_ils/importer/series/importer.py

```diff
@@ -120,7 +120,7 @@ def _normalize_title(title):
         # will become
         # `international series of numerical mathematics`
         for substring in IGNORE_SUFFIXES:
-            t = rreplace(t, substring, "")
+            t = rreplace_end(t, substring)
```

Comment:
I think it would be better to enhande rreplace function to use regex, as in
```
regex = substring + "$" # if i remember correctly this is the mark of the end of the string
if re.search(regex, series["title"]):
 series["title"] = rreplace(series["title"], regex, "")
```
instead of introducing a new parser function
please check also this occurrence:

https://github.com/CERNDocumentServer/cds-ils/commit/a7fd72e237e8ead29ad3a105d59d6949a192cc9a#diff-49df47ccd415564e19fca026f91495c2919495215eabea18c8bd00f31dd65049R88

Reply:
I think putting `Series` before `ser` in the list might solve this no? @anikachurilova